src/js/review/SubmitController.jsx

```diff
@@ -76,15 +73,11 @@ class SubmitController extends React.Component {
     // like to know if they’re common
     const { isValid, errors } = isValidForm(form, pagesByChapter);
     if (!isValid) {
-      recordEvent({
-        event: `${trackingPrefix}-validation-failed`,
-      });
-      Raven.captureMessage('Validation issue not displayed', {
-        extra: {
-          errors,
-          prefix: trackingPrefix
-        }
-      });
+      const recordEvent = formConfig.recordEvent ?
+        formConfig.recordEvent.bind(formConfig) :
```

Comment:
Yes this is a duplicate of the code in SubmitController. I was thinking that we might *not* want to log some messages that I want to add in the future (e.g. `{ event: 'user-changed-pages', from: 2, to: 3 }`) so it seemed bad to overwrite `formConfig.recordEvent` with a global default.